tst/working/quick/classical.tst

```diff
@@ -0,0 +1,20 @@
+#
+# tests for auxiliary functions in classical.gi
+#
+
+#
+# KroneckerFactors
+#
+gap> g:=[[1,2],[3,4]]*One(GF(5));;
+gap> h:=[[1,1],[3,-1]]*One(GF(5));;
+gap> KroneckerFactors(KroneckerProduct(g,h)) = [g,h];
+true
+
+# brute force test
+gap> G := SL(2,5);;
+gap> SetX(G, G, function(g,h)
+>   local gh;
+>   gh:=KroneckerFactors(KroneckerProduct(g,h));
+>   return gh = false or (IsEqualProjective(gh[1], g) and IsEqualProjective(gh[2], h));
+> end);
+[ true ]
```

Comment:
Always good to have dedicated tests for isolated functions like `KroneckerFactors` ! This promptly revealed a bug in the original version of this PR... oops.

Also makes for a handy benchmark. The above `SetX` call takes 650ms with this PR, down from 960ms before the PR. Not crucial but nice.